src/Assign/README.md

```diff
@@ -0,0 +1,140 @@
+# Assign
+```haskell
+Assign Object
+```
+`Assign` is a `Monoid` that will combine two objects using `Object.assign` on
```

Comment:
> `Assign` is a `Monoid` that will combine two objects using `Object.assign` on

Should stick with the following pattern:

`Assign` is a `Monoid` that will combine(2) objects under assignment using `Object.assign` on